circle.yml

```diff
@@ -55,9 +59,22 @@ test:
     # run tests on the emulator
     - ./gradlew :ReactAndroid:connectedAndroidTest -PdisablePreDex --stacktrace --info:
         timeout: 360
+
+    # testing docs generation is not broken
+    - cd website && node ./server/generate.js
   post:
     # copy test report for Circle CI to display
     - mkdir -p $CIRCLE_TEST_REPORTS/junit/
     - find . -type f -regex ".*/build/test-results/debug/.*xml" -exec cp {} $CIRCLE_TEST_REPORTS/junit/ \;
     - find . -type f -regex ".*/outputs/androidTest-results/connected/.*xml" -exec cp {} $CIRCLE_TEST_REPORTS/junit/ \;
 
+deployment:
+  website:
+    branch: [/.*-stable/, /master/]
+    commands:
+      # generate docs website
+      # TODO reach out to ben to get react-js user auth
```

Comment:
Ha, forgot to remove comment, already did

On Tuesday, 9 February 2016, Christopher Chedeau <notifications@github.com>
wrote:

> In circle.yml
> <https://github.com/facebook/react-native/pull/5760#discussion_r52358960>:
>
> >    post:
> >      # copy test report for Circle CI to display
> >      - mkdir -p $CIRCLE_TEST_REPORTS/junit/
> >      - find . -type f -regex ".*/build/test-results/debug/.*xml" -exec cp {} $CIRCLE_TEST_REPORTS/junit/ \;
> >      - find . -type f -regex ".*/outputs/androidTest-results/connected/.*xml" -exec cp {} $CIRCLE_TEST_REPORTS/junit/ \;
> >
> > +deployment:
> > +  website:
> > +    branch: [/.*-stable/, /master/]
> > +    commands:
> > +      # generate docs website
> > +      # TODO reach out to ben to get react-js user auth
>
> now is a good time :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/react-native/pull/5760/files#r52358960>.
>